cmake/modules/FindUMFPACK.cmake

```diff
@@ -27,10 +27,6 @@
 #   UMFPACK_VERSION_SUBMINOR
 #
 
-SET(UMFPACK_DIR "" CACHE PATH "An optional hint to an UMFPACK directory")
-SET(SUITESPARSE_DIR "" CACHE PATH
-  "An optional hint to a SUITESPARSE directory"
-  )
```

Comment:
I added a note to this line to the commit, it somehow disappeared. The problem is that if you set this here, the previous definition of UFMPACK_DIR is not used (I usually do what I believe a lot of people do - I have a CMake.vars file with all the definition and I include it at the beginning - pretty much like running cmake -D... -D...). I may be wrong.